.github/workflows/linux_packages_rpm.yaml

```
@@ -0,0 +1,289 @@
+          echo "✅ ARM64 RPM package verification passed"
+      
+      - name: Upload RPM artifact
+        uses: actions/upload-artifact@v4
+        with:
+          name: hyprnote-${{ env.VERSION }}-${{ matrix.arch }}.rpm
+          path: ${{ steps.find-rpm.outputs.rpm_path }}
+          if-no-files-found: error
+      
+      - name: Upload RPM to release
+        if: github.event_name == 'release'
+        uses: softprops/action-gh-release@v1
```

actionlint (1.7.8)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)